src/modules/user/user.controller.ts

```diff
@@ -95,3 +95,65 @@ userRouter.put(
     }
   },
 )
+
+userRouter.get(
+  '/:id',
+  TokenParserMiddleware,
+  IsUUID,
+  async (req: any, res: Response) => {
```

Comment:
what is primary purpose of this controller ? Users can't see each other's profiles ?

Reply:
made it visible for everyone